backend/ml/management/commands/ml_train.py

```diff
@@ -100,6 +101,18 @@ def save_data(video_scores, contributor_rating_scores, poll, trusted_only=True):
             for video_id, criteria, score, uncertainty in video_scores
         )
 
+        entities = []
+        for entity in (
+            Entity.objects.filter(criteria_scores__poll=poll)
+            .distinct()
+            .prefetch_related("criteria_scores")
+        ):
+            entity.tournesol_score = 10 * sum(
+                criterion.score for criterion in entity.criteria_scores.iterator()
```

Comment:
After running "ml_train" with debug messages about SQL queries, I can see that this `.iterator()` too would cancel the effect of `prefetch_related()`:
```suggestion
                criterion.score for criterion in entity.criteria_scores.all()
```